### ISSUE TITLE
LOBSTER parser fails

### ISSUE THREAD

**Issue by naik-aakash** [nomad-coe/workflow-parsers#5]
Hi @markus1978

 I Tried uploading  lobster calculation output files and Parser seems to fail. Any idea why it is happening? Can provide you the upload ID if needed. Thanks in advance.

I am at the moment trying to see, If I encounter similar problem locally.

**Comment by naik-aakash** [comment]
> Hi @markus1978
> 
> I Tried uploading lobster calculation output files and Parser seems to fail. Any idea why it is happening? Can provide you the upload ID if needed. Thanks in advance.
> 
> I am at the moment trying to see, If I encounter similar problem locally.

The issue is parser does not support lobster 4.1.0 output files which have orbital resolved COHP, COOP, ICOHPLIST, ICOOPLIST etc. Will try to implement support to this version. 

I hope this is okay 😄

**Comment by ondracka** [comment]
@naik-aakash  could you please share the testcase? There are some Lobster 4.1.0 entries in NOMAD (example: https://nomad-lab.eu/prod/v1/staging/gui/search/entries/entry/id/fK-sxuN9uKrEZNK8saEuvTw99U0E) and they seem to be parsed fine.

**Comment by naik-aakash** [comment]
@ondracka , Yes, I can share the test case. I just quickly checked link for entry on NOMAD you provided. I see these LOBSTER calculation files do not contain orbital resolved information and as the output format of these files are similar to LOBSTER 4.0 output, they seem to parse. 
Here is the link for my test upload:
https://nomad-lab.eu/prod/v1/staging/api/v1/entries/MvYh4OWtlJGvHbD8GP2q1CqcZ4sA

**Comment by ondracka** [comment]
Is it published? I get "The entry with the given id does not exist or is not visible to you.". I'm the original author of the Lobster parser and an occasional contributor, but I'm not part of the NOMAD teams, so I don't have access to unpublished stuff.

**Comment by naik-aakash** [comment]
Ohh okay, and yes it is not published and it is just one of the several compuations. I was thinking of uploading here if parsers worked and then publish. 

I can provide you the key difference in the lobsterin file compared to example case you provided. This `orbitalwise` keyword alters the file format of version 4.1 and then parsers fail. Maybe  you can try it out for some of your exisiting computation? 


cohpGenerator from 0.1 to 6.0 orbitalwise

**Comment by ondracka** [comment]
OK I see the difference, if you need a quick fix I can try to disable the COHP/COOP parsing in this case for now so that it doesn't crash and at least the rest of the properties is parsed. Implementing support for the new format will take a bit...

**Comment by naik-aakash** [comment]
Hi, thank you for the help and offer. There is no rush in this regard. I had already started work on it a bit. It needs some more refinements, but it manages to parse COHP files and so on. Will try to raise a PR soon and then you could review it. 😄
